infra/helper.py

```diff
@@ -278,17 +278,29 @@ def get_dockerfile_path(project_name):
 
 def _get_corpus_dir(project_name=''):
   """Returns path to /corpus directory for the given project (if specified)."""
-  return os.path.join(BUILD_DIR, 'corpus', project_name)
+  path = os.path.join(BUILD_DIR, 'corpus', project_name)
```

Comment:
I don't think 'corpus' is guaranteed to exist. So how about you use
`os.makedirs(directory, exist_ok=True)`.
This will make the parent directory and remove the need for checking the directory exists.

Also, could you please update the docstrings of these functions to reflect the fact that they create the directories if they don't already exist?